pages/_app.tsx

```diff
@@ -11,9 +11,25 @@ import { firebaseClient } from "../utils/firebaseClient";
 require("../styles/globals.css");
 require("@solana/wallet-adapter-react-ui/styles.css");
 
+const auth = getAuth(firebaseClient);
+async function callback(data: Record<string, string>) {
+  const userCredential = await signInWithCustomToken(auth, data.token);
+  return userCredential;
+}
+
+const signOut = () => {
+  return auth.signOut();
+};
+
 const App: FC<AppProps> = ({ Component, pageProps }) => {
   return (
-    <SolanaAuthProvider>
+    <SolanaAuthProvider
+      domain="desolate.space"
```

Comment:
WDYT of puttng put `<your domain goes here>` or something like that

Reply:
I plan to make those changes as I see what needs to be done to turn it into a package, but happy to make then now too.

Reply:
ah let's merge then